kbdcfg.lua

```diff
@@ -95,6 +116,8 @@ local function factory(args)
     kbdcfg.current               = args.current or 1
     kbdcfg.menu                  = nil
     kbdcfg.type                  = args.type or "tui"
+    kbdcfg.remember_layout       = args.remember_layout or false
+    kbdcfg.default_layout_index  = args.default_layout_index or 1
```

Comment:
Could you please move it before `kbdcfg.current` and use this value as a default value for `kbdcfg.current`?